magento/pwa-studio
# Pwa 1871

# Description

**Who** As a user,
**What** I want the Wishlist Edit Dialog errors to clear when I close and re-open the dialog
**Why** so that I don't get confused

To reproduce:

Navigate to the `/wishlist` page
Click the "..." to open the "List Actions" dialog
Click "Edit List"
Trigger an error - one easy way is to disable multiple wishlist configuration on your backend
Close the dialog
Re-open the "List Actions" and then "Edit List" dialog
See that the previous form error still appears
Expected: Upon re-opening the dialog, the previous form errors should no longer be present.

Developer Notes: Fixed all the typos that spelled "wish" with "whis"

## Related Issue







Closes [#PWA-1871](https://jira.corp.magento.com/browse/PWA-1871)

## Acceptance





### Verification Stakeholders



### Specification




## Verification Steps
Login to user with 2 wishlist
Navigate to the `/wishlist` page
Click the "..." to open the "List Actions" dialog
Click "Edit List"
Try to rename first wishlist to same name as second wishlist
Get error
Close the dialog
Re-open the "List Actions" and then "Edit List" dialog
See that the previous form error is gone


#### Test scenario(s) for direct fix/feature


1. Verify action menu dialog is no longer persist between closing and opening dialog
2. Make sure ... icon should not render on EE if multiple wishlist is disabled

#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
